exist-core/src/main/java/org/exist/http/RESTServer.java

```diff
@@ -1162,61 +1191,66 @@ public void doDelete(final DBBroker broker, final Txn transaction, final String
 
     private boolean checkForXQueryTarget(final DBBroker broker, final Txn transaction,
         final XmldbURI path, final HttpServletRequest request,
-        final HttpServletResponse response) throws PermissionDeniedException,
-        NotFoundException, IOException, BadRequestException {
-        
+        final HttpServletResponse response) throws PermissionDeniedException, IOException, BadRequestException {
```

Comment:
NotFoundException is gone?

Reply:
yep

Reply:
It made no sense so I was wondering why it was there and then Idea also told me it was not needed.